### PR TITLE
[Delta] Removes SSU From Mining Equipment Room

### PR DESCRIPTION
**Pull request by JohnTheDonut**
## **Removes SSU From Mining Equipment Room On Deltastation**
Removes the three Suit Units which provide basic space suits in the mining equipment room on Deltastation. This is because now with Lavaland using a space suit is a pretty big death sentence because of the movement speed decrease, but more importantly it provides traitor miners with a unnecessary buff and an extremely easy start along with the other equipment they have. This Pull Request should make traitor miner a bit more challenging on Deltastation and even it out compared to other maps which do not start with miner SSU's, this should also decrease the amount of break ins into cargo/mining by greytiders to get space suits. 

## **Changelog**
:cl: Tofa01
Del: [Delta] Removes SSU From Mining Equipment Room
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
Good work Tofa

**Comment by Tofa01** [comment]
Thanks Oranges :)

**Comment by jughu** [comment]
And slowy delta be another reskin of box....

**Comment by Repukan** [comment]
Yey they read my suggestion

**Comment by Metallusions** [comment]
>praise from Repukan and oranges on the same pr
That's how you know you're making a huge mistake.

**Comment by Repukan** [comment]
Alright then tell us how providing hardsuit with major slowdowns that get new miners killed is good.

Or tell us how giving traitor miners almost 6 TC of free equipment roundstart is balanced.

Or, how providing greytiders with no access requirement hardsuits (sure hidden behind two doors that you know they'll hack) is good.

**Comment by Dorsisdwarf** [comment]
that sure sounds like a huge problem which is detrimental to all rounds, these mythical greytiders who steal these hardsuits every round even though the engineering hardsuits are also no-access 2-doors from assistant's grubby hands, as is the captain's, and the CMOs, and on some maps the RD's. Oh and all the EVA suits are 1 door from maint.


In fact, the only spacesuits further than 2 doors from a public access on box that I can think of are the security hardsuits. Not the HOS' though, that's two doors from public maint too.

On the other hand why on earth would you put mining hardsuits on the map, lavaland killed their actual use and replaced it with a much better system.

**Comment by Metallusions** [comment]
1. new miners are going to die against anything that requires full mobility anyway (megafauna) - tell them to use the SSUs on the lavaland base instead if you value their lives so much
2. EVA already exists
3. EVA already exists

**Comment by SgtxRootbeer** [comment]
Cause sometimes, I dare to get a red slime potion, or I want to escape the singuloose that the assistants who dared to hack 3 airlocks set loose.
Again, jughu is right for once.

**Comment by XDTM** [comment]
There's other ways to not be Box than powercreeping.

**Comment by Kromgar** [comment]
The extra suits are because delta is a high pop station. If your removing the hardsuits replace them with 3 more explorer suits

**Comment by Mindustry** [comment]
@Kromgar 
That argument is really weak, you don't even need a suit to go out onto lavaland. You certainly do not need a space worthy hardsuit either. 

There is no need to add more explorer suits than miner slots either. Anyone additional can easily order starter miner gear from either asking a miner to give them a conscription kit, or using 2500 points (like four plasma ore?) to order basically the same minus the free access card. You could also just go down to lavaland and make bone armor.

The bottom line is that if you want a mining hardsuit, you can purchase one using your mining points from the mining vendor. We are past the days of asteroid. There's no reason to give them out for free.

**Comment by Kromgar** [comment]
@Mindustry most miners have already used their voucher in the 1st 30 seconds most dont bother getting conscription kits.

>2500 pointsi n 4 plasma ore


Your so fucking retarded i swear to fucking god.

No thats about 83 plasma ore dipshit.


Also the main reason a miner needs an explorer suit is for FIRE protection.

**Comment by Mindustry** [comment]
@Kromgar 
Alright fucko, if we're going to be fucking hostile about it. What I said was four ore, it's actually FIVE.

One ore = 500 credits you dumb shit. Do you not know cargo?

**Comment by jughu** [comment]
83 plasma ore is exactly 41,500 now put that trough the orm upgrade multiplier

**Comment by Tofa01** [comment]
Guys.... Lavaland already has a few SSU's if miners for whatever reason want a Space Suit there are some there i don't see the reason to add an extra three free suits on the station. If you want to obtain a space suit as someone who really doesn't need a space suit it should not be as easy as 1. Get mining access 2. Walk in front room 3.Get space suit. you should need to get a higher access such as EVA access or be able to take a risk if you really need the suit and have to break into EVA (The AI also exists).

**Comment by Mindustry** [comment]
@Tofa01 
> Lavaland already has a few SSU
Those have the explorer suits in them. Miners are not supposed to be getting free space suits, this change is good. Unless I'm mistaken and Delta, for whatever reason, uses a custom lavaland.

**Comment by Tofa01** [comment]
I was not aware that those SSU's contained explorer suits, though my other point still stands if in the end miners really need space suits they should be added to lavaland not the station itself.

**Comment by Mindustry** [comment]
@Tofa01 
No, they shouldn't be added at all to either station or lavaland. Miners don't need them at all roundstart and they can already be bought in the mining vendors.

**Comment by Tofa01** [comment]
I see, that would be another point of this PR, why should miners get free ones at roundstart when they can be already bought.

**Comment by Hatfish** [comment]
Just mine a bit for once in your lives and buy a hardsuit for 2000 points, you nerds.

**Comment by Repukan** [comment]
One plasma ore can get you enough plasma tech levels to print on a disk, ship it, and order the mining starter kits.

Literally no excuse for keeping the hardsuits.
